packages/hello-gsm-admin/src/components/ContentBox/index.tsx

```diff
@@ -56,134 +57,91 @@ const ContentBox: React.FC<ContentType> = ({
-      if (error.response.status === 401) {
-        try {
-          // accessToken 발급
-          await auth.refresh();
-          documentSubmission();
-        } catch (error) {
-          console.log(error);
-        }
-      } else {
-        console.log(error);
-      }
-    }
-  };
-
   const resultStyle = (result: '미정' | '합격' | '불합격') => {
     switch (result) {
       case '미정':
         return css`
-          background: #625e6f;
-          color: rgba(31, 31, 31, 0.86);
+          color: #9e9e9e;
           cursor: default;
         `;
       case '합격':
         return css`
-          background: #19baff;
-          color: #ffffff;
+          color: #2174d8;
           cursor: default;
         `;
       case '불합격':
         return css`
-          background: #ff4747;
-          color: #ffffff;
+          color: #ff000f;
           cursor: default;
         `;
     }
   };
 
-  const scoreButtonClick = () => {
-    if (
-      // 2차 성적을 입력하는 기간
-      isDuringFinalResult
-    ) {
-      return toast.error('2차 성적 입력 기간이 아닙니다.');
-    }
-    if (firstResult === '불합격') {
-      return toast.error('불합격자는 2차 성적 입력이 불가능합니다.');
-    }
-    setModalRegistrationNumber(registrationNumber);
-    setModalName(name);
-    setShowScoreModal();
-    setScoreModalValue(score);
-  };
-
   return (
     <S.ContentBox>
       <S.Content>
-        <S.RegistrationNumber>{applicationIdx}</S.RegistrationNumber>
+        <S.RegistrationNumber>
+          {String(applicationIdx).padStart(4, '0')}
+        </S.RegistrationNumber>
+        <S.isDocumentReception>
+          <S.DocumentReceptionText documentReception={documentReception}>
+            {documentReception ? '· 제출' : '· 미제출'}
+          </S.DocumentReceptionText>
+        </S.isDocumentReception>
         <S.Name>{name}</S.Name>
         <S.Screening>{screening}</S.Screening>
         <S.SchoolName>
           {schoolName !== 'null' ? schoolName : '검정고시'}
         </S.SchoolName>
-        <S.isDocumentReception>
-          <S.Checkbox
-            css={css`
-              background: ${documentReception && '#19BAFF'};
-            `}
-            onClick={documentSubmission}
-          />
-        </S.isDocumentReception>
-        <S.PhoneNumber>{cellphoneNumber}</S.PhoneNumber>
-        <S.GuardianNumber>{guardianCellphoneNumber}</S.GuardianNumber>
-        <S.TeacherNumber>
-          {teacherCellphoneNumber !== 'null'
-            ? teacherCellphoneNumber
-            : '검정고시'}
-        </S.TeacherNumber>
+        <S.PhoneNumber>{formattedCellphoneNumber}</S.PhoneNumber>
+        <S.GuardianNumber>{formattedGuardianCellphoneNumber}</S.GuardianNumber>
+        <S.TeacherNumber>{formattedTeacherCellphoneNumber}</S.TeacherNumber>
+        <S.FirstResultText css={() => resultStyle(firstResult)}>
+          {firstResult}
+        </S.FirstResultText>
+        <S.FinalScoreText
+          css={() => {
+            return { color: score ? '#212121' : '#9E9E9E' };
+          }}
```

Comment:
```suggestion
          css={css`
            color: ${score ? '#212121' : '#9E9E9E'};
          `}
```

css``문법을 이용해서 간결하게 줄힐 수 있을 것 같습니다.

Reply:
넵 반영하겠습니다!!